dualis_connector/dualis_service.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, config_helper: ConfigHelper):
 
         self.is_state_floating = False
 
-    def interactively_acquire_token(self) -> str:
```

Comment:
I'd find it cleaner to keep this as two methods:
`interactively_aquire_token()` gives the command line prompts and than, given they are successful, vcalls `aquire_token(mail, pw)`.
Accordingly, when calling the CLI for `--new-token` with a file, it would call directly `aquire_token(mail, pw)`